lessons/static/css/commonlesson.css

```diff
@@ -383,12 +383,10 @@ html, body{
 }
 
 .tiplink {
-    float: left;
     width: 20px;
     font-size: 15px;
-    margin-left: -20px;
     clear: left;
-    display: block;
+    display: inline;
```

Comment:
we could probably get rid of this property altogether, as spans default to displaying inline